packages/expo-updates/ios/EXUpdates/AppLoader/EXUpdatesFileDownloader.h

```diff
@@ -22,9 +23,10 @@ typedef void (^EXUpdatesFileDownloaderErrorBlock)(NSError *error);
                  errorBlock:(EXUpdatesFileDownloaderErrorBlock)errorBlock;
 
 - (void)downloadFileFromURL:(NSURL *)url
+              verifyingHash:(nullable NSString *)expectedBase64URLEncodedSHA256Hash
                      toPath:(NSString *)destinationPath
```

Comment:
I like the way the method name conveys that the file is downloaded -> hash is verified -> file is saved to path in that order